modules/environment/aws/github-runners-s3/main.tf

```diff
@@ -49,7 +53,7 @@ resource "aws_iam_policy" "github_runners" {
                 "s3:GetBucketVersioning",
                 "s3:CreateBucket"
      ],
-     "Resource": ["arn:aws:s3:::github-runners-${data.aws_caller_identity.current.account_id}-${var.cluster_name}.liatr.io"]
+     "Resource": ["arn:aws:s3:::${local.bucket_name}"]
```

Comment:
this arn string could be replaced with `aws_s3_bucket.github-runner.arn` so we don't have to build the string ourselves.